cumulusci/tasks/salesforce/tests/test_Deploy.py

```diff
@@ -43,10 +44,356 @@ def test_get_api__managed(self):
+            )
+
+            # non_lwc_component_directories
+            for d in non_lwc_component_directories:
+                self.assertEqual(True, task._include_file(d, "file_name" + file_ending))
+
+        # file endings not in lwc component whitelist
+        for file_ending in ["", ".json", ".xml", ".cls", ".cls-meta.xml", ".object"]:
+            # lwc_component_directory
+            self.assertEqual(
+                False,
+                task._include_file(lwc_component_directory, "file_name" + file_ending),
+            )
+
+            # non_lwc_component_directories
+            for d in non_lwc_component_directories:
+                self.assertEqual(True, task._include_file(d, "file_name" + file_ending))
+
+    def test_get_files_to_package(self):
+        with temporary_dir() as path:
+            expected = []
+
+            # add package.xml
+            with open(os.path.join(path, "package.xml"), "w") as f:
+                f.write(
+                    """<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <version>45.0</version>
+</Package>"""
+                )
+                expected.append(os.path.join(".", "package.xml"))
+
+            # add lwc
+            lwc_path = os.path.join(path, "lwc")
+            os.mkdir(lwc_path)
+
+            # add lwc linting files (not included in zip)
+            lwc_ignored_files = [".eslintrc.json", "jsconfig.json"]
+            for lwc_ignored_file in lwc_ignored_files:
+                with open(os.path.join(lwc_path, lwc_ignored_file), "w"):
+                    pass
+
+            # add lwc component
+            lwc_component_path = os.path.join(lwc_path, "myComponent")
+            os.mkdir(lwc_component_path)
+
+            # add lwc component files included in zip (in alphabetical order)
+            lwc_component_files = [
+                {"name": "myComponent.html"},
+                {"name": "myComponent.js"},
+                {
+                    "name": "myComponent.js-meta.xml",
+                    "body:": """<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="myComponent">
+    <apiVersion>45.0</apiVersion>
+    <isExposed>false</isExposed>
+</LightningComponentBundle>""",
+                },
+                {"name": "myComponent.svg"},
+                {"name": "myComponent.css"},
+            ]
+            for lwc_component_file in lwc_component_files:
+                with open(
+                    os.path.join(lwc_component_path, lwc_component_file.get("name")),
+                    "w",
+                ) as f:
+                    if lwc_component_file.get("body") is not None:
+                        f.write(lwc_component_file.get("body"))
+                    expected.append(
+                        os.path.join(
+                            "./lwc/myComponent", lwc_component_file.get("name")
```

Comment:
Separate out the path components and pass them to os.path.join so you get the right separator instead of always /

Reply:
Yes.  I saw the test failure on a windows build.  This is done.